content/licenses_gov.qmd

```diff
@@ -8,17 +8,15 @@ Work of the United States government that is done by US federal employees as par
 
 Government agencies will use some kind of public domain license, but which one used varies. There are two main licenses used for products by federal agencies (or their employees) on GitHub: Creative Commons license (CC0-1.0) and the GNU General Public License v3. [CC0-1.0 is a very broad declaration of public domain](https://creativecommons.org/publicdomain/zero/1.0/) while [GPL-3 is explicit about derivative works and how those works must retain an open source license](https://www.gnu.org/licenses/gpl-3.0.en.html).
 
-<!--
-* The [NOAA Fisheries Integrated Toolbox](https://noaa-fisheries-integrated-toolbox.github.io/) recommends [GPL-3](https://github.com/nmfs-fish-tools/Resources/blob/master/LICENSE) over CC0-1.0 since the latter is not designed specifically for software. An [additional note](https://github.com/nmfs-fish-tools/Resources/blob/master/LICENSE.md) is added to clarify that the material is a US Government work. See their [Resources](https://github.com/nmfs-fish-tools/Resources) repository.
--->
+Special comments re licenses for open source software. CC0 [does not recommend the license for software](https://creativecommons.org/faq/#can-i-apply-a-creative-commons-license-to-software) but instead recommend using an open source license designed for software. If you want to publish your tool in the Journal of Open Source Software, for example, CC0 is not acceptable because it is not one of the [listed open source licenses](https://opensource.org/licenses/) on the Open Source Initiative.
```

Comment:
Looks good! Perhaps some minor changes:
```
There are some special considerations for licenses for open source software. The CC0 website [does not recommend the license for software](https://creativecommons.org/faq/#can-i-apply-a-creative-commons-license-to-software), but instead recommends using an open source license designed for software. If you want to publish your tool in the [Journal of Open Source Software](https://joss.theoj.org/), for example, CC0 is not acceptable because it is not one of the [listed open source licenses](https://opensource.org/licenses/) on the Open Source Initiative.
```

Reply:
Looks good. Added these changes.